src/ui/server/controllers/insurance/policy/loss-payee/save-and-back/index.ts

```diff
@@ -0,0 +1,49 @@
+import { ROUTES } from '../../../../../constants';
+import constructPayload from '../../../../../helpers/construct-payload';
+import generateValidationErrors from '../../../../../shared-validation/yes-no-radios-form';
+import mapAndSave from '../../map-and-save/nominated-loss-payee';
+import { FIELD_ID, ERROR_MESSAGE } from '..';
+import { Request, Response } from '../../../../../../types';
+
+const { INSURANCE_ROOT, ALL_SECTIONS, PROBLEM_WITH_SERVICE } = ROUTES.INSURANCE;
+
+/**
+ * saves and goes back to all sections from loss payee page unless there are API errors
+ * @param {Express.Request} Express request
+ * @param {Express.Response} Express response
+ * @returns {Express.Response.redirect} redirects to all sections page on success
+ */
+const post = async (req: Request, res: Response) => {
+  try {
+    const { application } = res.locals;
+
+    const { referenceNumber } = req.params;
+
+    if (!application) {
+      return res.redirect(PROBLEM_WITH_SERVICE);
+    }
+
+    const { body } = req;
+
+    const payload = constructPayload(body, [FIELD_ID]);
+
+    // run validation on inputs
+    const validationErrors = generateValidationErrors(payload, FIELD_ID, ERROR_MESSAGE);
+
+    // runs save and go back command
+    const saveResponse = await mapAndSave.nominatedLossPayee(payload, application, validationErrors);
+
+    if (!saveResponse) {
+      return res.redirect(PROBLEM_WITH_SERVICE);
+    }
+
+    // redirect to all sections page
```

Comment:
I don't think we need such comments in `save-and-back`, most places don't have this. Feels self explanatory without to me, what do you think?

Reply:
removed